packages/relay/src/lib/eth.ts

```diff
@@ -1720,9 +1721,16 @@ export class EthImpl implements Eth {
     if (submittedTransactionId) {
       try {
         const formattedTransactionId = formatTransactionIdWithoutQueryParams(submittedTransactionId);
+
+        // Create a modified copy of requestDetails
+        const modifiedRequestDetails = {
```

Comment:
The variable is used only once so we can skip its declaration and pass the object straight to the `repreatRequest` method.